meta-refkit/recipes-image/images/initramfs-framework-refkit-dm-verity.bb

```diff
@@ -0,0 +1,112 @@
+# This recipe creates a module for the initramfs-framework in OE-core
+# which opens the partition identified via the "root"
+# kernel parameter using the dm-verity hashes stored in
+# the partition identified via the "dmverity" kernel
+# parameter and changes bootparam_root so
+# that the following init code uses the read-only,
+# integrity protected mapped partition.
+
+SUMMARY = "dm-verity module for the modular initramfs system"
+LICENSE = "MIT"
+DEPENDS = "openssl-native"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
+
+require refkit-boot-settings.inc
+
+refkit_dmverity[shellcheck] = "sh"
+refkit_dmverity () {
+
+    dmverity_enabled() {
+        [ "$bootparam_dmverity" ]
+    }
+
+    dmverity_run () {
+        C=0
+        delay=${bootparam_rootdelay:-1}
```

Comment:
initramfs-framework doesn't use that parameter elsewhere either. It might make sense to allow waiting indefinitely (because the fallback is to give up), but that should be introduced across the board, not just here.